nixos/modules/programs/goldwarden.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, config, pkgs, ... }:
+let
+  cfg = config.programs.goldwarden;
+in
+{
+  options.programs.goldwarden = {
+    enable = lib.mkEnableOption "Goldwarden";
+    package = lib.mkPackageOption pkgs "goldwarden" {};
+  };
+
+  config = lib.mkIf cfg.enable {
+    environment.systemPackages = [
+      # for cli and polkit action
+      cfg.package
+      # binary exec's into pinentry which should match the DE
+      pkgs."pinentry-${config.programs.gnupg.agent.pinentryFlavor}"
```

Comment:
This doesn't work right now (e.g. with gnome3), are we waiting for another PR ?

Reply:
Yes, this requires #133542